app/design/frontend/base/default/layout/captcha.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<layout version="0.1.0">
+    <default>
+        <reference name="head">
+            <action method="addJs"><name>maho-captcha.js</name></action>
```

Comment:
I understand the rationale but I don't know... it's another http request which then loads the altcha script, IMHO we should have avoided another external js :-\

Reply:
Hm, but isn't it better to serve an external JS and cache it instead of putting inline JS in the HTML body? The latter adds bytes to every request instead of just once.

Reply:
if the JS was short short then to me it's better to have it inline, if become too big (but big js come with another set of problems :-() then external

Reply:
> it's another http request which then loads the altcha script

Also, I think that's okay because the altcha script doesn't need to be high priority. It just needs to be loaded by the time the user clicks into a form element.

Reply:
we've no way of setting priority to `<script src="http://maho.test/js/maho-captcha.js"></script>` like defer/async right? cause this specific script would need that

Reply:
I meant more so that if altcha.min.js wasn't loaded via XML, that its loading would be deferred via the JS focusin event. We should have a way to set those script attributes via XML, but we don't currently as far as I know.

But is your idea to output directly to the `<head>` something like this?:
```html
<head>
<script src="http://maho.test/js/maho-captcha.js" defer></script>
<script>
document.addEventListener('DOMContentLoaded', () => {
    MahoCaptcha.setup({ ... });
});
</script>
</head>
```

Reply:
I did it in https://github.com/MahoCommerce/maho/pull/109/commits/69754ff7d1fe12f3a1f8df8f6fb1c3539d447982, sorry but I've changed the "if helper->enabled" check again